### PR TITLE
Update 2017 blog content files to move author details in front-matter

### PR DESCRIPTION
**Pull request by dipesh-rawat**
Split out from PR https://github.com/kubernetes/website/pull/45957

This PR extends the changes made in PR https://github.com/kubernetes/website/pull/45865 and implements those changes to all the 2018 blog files as per the recommendation in https://github.com/kubernetes/website/pull/45865#issuecomment-2054036103. The primary change involves relocating author details into the metadata within the front matter.

Useful Link
[Preview Blog Summary Page](https://deploy-preview-46627--kubernetes-io-main-staging.netlify.app/blog)

### CODE REVIEW

File: content/en/blog/_posts/2017-02-00-Caas-The-Foundation-For-Next-Gen-Paas.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: " Containers as a Service, the foundation for next generation PaaS "
 date: 2017-02-21
 slug: caas-the-foundation-for-next-gen-paas
 url: /blog/2017/02/Caas-The-Foundation-For-Next-Gen-Paas
+author: >
+  [Brendan Burns](https://twitter.com/brendandburns) (Microsoft and Co-founder of Kubernetes)
```
Comment (sftim) [inline content/en/blog/_posts/2017-02-00-Caas-The-Foundation-For-Next-Gen-Paas.md:7]:
```suggestion
  [Brendan Burns](https://twitter.com/brendandburns) (Microsoft)
```

File: content/en/blog/_posts/2017-02-00-Caas-The-Foundation-For-Next-Gen-Paas.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ title: " Containers as a Service, the foundation for next generation PaaS "
...
 ---
 
 _Today’s post is by Brendan Burns, Partner Architect, at Microsoft & Kubernetes co-founder._
```
Comment (sftim) [inline content/en/blog/_posts/2017-02-00-Caas-The-Foundation-For-Next-Gen-Paas.md:10]: I'd drop or revise this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-02-00-Run-Deep-Learning-With-Paddlepaddle-On-Kubernetes.md [hunk 3]
```diff
@@ -3,6 +3,9 @@ title: " Run Deep Learning with PaddlePaddle on Kubernetes "
 date: 2017-02-08
 slug: run-deep-learning-with-paddlepaddle-on-kubernetes
 url: /blog/2017/02/Run-Deep-Learning-With-Paddlepaddle-On-Kubernetes
+author: >
+  Yi Wang ([Baidu Research](http://research.baidu.com/)),
+  Xiang Li ([CoreOS](https://coreos.com/))
 ---
 
 _Editor's note: Today's post is a joint post from the deep learning team at Baidu and the etcd team at CoreOS._
```
Comment (sftim) [inline content/en/blog/_posts/2017-02-00-Run-Deep-Learning-With-Paddlepaddle-On-Kubernetes.md:11]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-03-00-K8Sport-Engaging-The-Kubernetes-Community.md [hunk 4]
```diff
@@ -3,6 +3,8 @@ title: " The K8sPort: Engaging Kubernetes Community One Activity at a Time "
 date: 2017-03-24
 slug: k8sport-engaging-the-kubernetes-community
 url: /blog/2017/03/K8Sport-Engaging-The-Kubernetes-Community
+author: >
+   Ryan Quackenbush (Apprenda) 
 ---
 _Editor's note: Today’s post is by Ryan Quackenbush, Advocacy Programs Manager at Apprenda, showing a new community portal for Kubernetes advocates: the K8sPort._
```
Comment (sftim) [inline content/en/blog/_posts/2017-03-00-K8Sport-Engaging-The-Kubernetes-Community.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-03-00-Kubernetes-1-6-Multi-User-Multi-Workloads-At-Scale.md [hunk 5]
```diff
@@ -3,6 +3,8 @@ title: " Kubernetes 1.6: Multi-user, Multi-workloads at Scale "
 date: 2017-03-28
 slug: kubernetes-1.6-multi-user-multi-workloads-at-scale
 url: /blog/2017/03/Kubernetes-1-6-Multi-User-Multi-Workloads-At-Scale
+author: >
+  Aparna Sinha (Google)
 ---
 Today we’re announcing the release of Kubernetes 1.6.
```
Comment (sftim) [inline content/en/blog/_posts/2017-03-00-Kubernetes-1-6-Multi-User-Multi-Workloads-At-Scale.md:9]:
```suggestion
_This article is by Aparna Sinha on behalf of the Kubernetes 1.6 release team._

Today we’re announcing the release of Kubernetes 1.6.
```
(justified by the “we”)

File: content/en/blog/_posts/2017-04-00-Multi-Stage-Canary-Deployments-With-Kubernetes-In-The-Cloud-Onprem.md [hunk 6]
```diff
@@ -3,6 +3,8 @@ title: " How Bitmovin is Doing Multi-Stage Canary Deployments with Kubernetes in
 date: 2017-04-21
 slug: multi-stage-canary-deployments-with-kubernetes-in-the-cloud-onprem
 url: /blog/2017/04/Multi-Stage-Canary-Deployments-With-Kubernetes-In-The-Cloud-Onprem
+author: >
+  Daniel Hoelbling-Inzko (Bitmovin)
 ---
 _Editor's Note: Today’s post is by Daniel Hoelbling-Inzko, Infrastructure Architect at Bitmovin, a company that provides services that transcode digital video and audio to streaming formats, sharing insights about their use of Kubernetes._
```
Comment (sftim) [inline content/en/blog/_posts/2017-04-00-Multi-Stage-Canary-Deployments-With-Kubernetes-In-The-Cloud-Onprem.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-05-00-Draft-Kubernetes-Container-Development.md [hunk 7]
```diff
@@ -3,6 +3,8 @@ title: " Draft: Kubernetes container development made easy "
 date: 2017-05-31
 slug: draft-kubernetes-container-development
 url: /blog/2017/05/Draft-Kubernetes-Container-Development
+author: >
+  Brendan Burns (Microsoft Azure)
 ---
 _Today's post is by __Brendan Burns, Director of Engineering at Microsoft Azure and Kubernetes co-founder._
```
Comment (sftim) [inline content/en/blog/_posts/2017-05-00-Draft-Kubernetes-Container-Development.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-05-00-Kubernetes-Monitoring-Guide.md [hunk 8]
```diff
@@ -3,6 +3,8 @@ title: " Kubernetes: a monitoring guide "
 date: 2017-05-19
 slug: kubernetes-monitoring-guide
 url: /blog/2017/05/Kubernetes-Monitoring-Guide
+author: >
+  Jean-Mathieu Saponaro (Datadog)
 ---
 _Today’s post is by Jean-Mathieu Saponaro, Research & Analytics Engineer at Datadog, discussing what Kubernetes changes for monitoring, and how you can prepare to properly monitor a containerized infrastructure orchestrated by Kubernetes._
```
Comment (sftim) [inline content/en/blog/_posts/2017-05-00-Kubernetes-Monitoring-Guide.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-05-00-Kubernetes-Security-Process-Explained.md [hunk 9]
```diff
@@ -3,6 +3,9 @@ title: " Dancing at the Lip of a Volcano: The Kubernetes Security Process - Expl
 date: 2017-05-18
 slug: kubernetes-security-process-explained
 url: /blog/2017/05/Kubernetes-Security-Process-Explained
+author: >
+  Brandon Philips (CoreOS),
+  Jess Frazelle (Google)
 ---
 _Editor's note: Today’s post is by&nbsp; __Jess Frazelle of Google and Brandon Philips of CoreOS about the Kubernetes security disclosures and response policy.__ &nbsp;_
```
Comment (sftim) [inline content/en/blog/_posts/2017-05-00-Kubernetes-Security-Process-Explained.md:10]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-05-00-Kubespray-Ansible-Collaborative-Kubernetes-Ops.md [hunk 10]
```diff
@@ -3,6 +3,8 @@ title: " Kubespray Ansible Playbooks foster Collaborative Kubernetes Ops "
 date: 2017-05-19
 slug: kubespray-ansible-collaborative-kubernetes-ops
 url: /blog/2017/05/Kubespray-Ansible-Collaborative-Kubernetes-Ops
+author: >
+   Rob Hirschfeld (RackN)
 ---
 _Today’s guest post is by Rob Hirschfeld, co-founder of open infrastructure automation project, Digital Rebar and co-chair of the SIG Cluster Ops. &nbsp;_
```
Comment (sftim) [inline content/en/blog/_posts/2017-05-00-Kubespray-Ansible-Collaborative-Kubernetes-Ops.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-06-00-Kubernetes-1-7-Security-Hardening-Stateful-Application-Extensibility-Updates.md [hunk 11]
```diff
@@ -3,6 +3,9 @@ title: " Kubernetes 1.7: Security Hardening, Stateful Application Updates and Ex
 date: 2017-06-30
 slug: kubernetes-1.7-security-hardening-stateful-application-extensibility-updates
 url: /blog/2017/06/Kubernetes-1-7-Security-Hardening-Stateful-Application-Extensibility-Updates
+author: >
+   Aparna Sinha (Google),
+   Ihor Dvoretskyi (Mirantis)
 ---
 Today we’re announcing Kubernetes 1.7, a milestone release that adds security, storage and extensibility features motivated by widespread production use of Kubernetes in the most demanding enterprise environments.&nbsp;
```
Comment (sftim) [inline content/en/blog/_posts/2017-06-00-Kubernetes-1-7-Security-Hardening-Stateful-Application-Extensibility-Updates.md:10]: I'd write this:
```suggestion
_This article is by Aparna Sinha and Ihor Dvoretskyi, on behalf of the Kubernetes 1.7 release team._

Today we’re announcing Kubernetes 1.7, a milestone release that adds security, storage
and extensibility features motivated by widespread production use of Kubernetes
in the most demanding enterprise environments.
```

File: content/en/blog/_posts/2017-08-00-Kompose-Helps-Developers-Move-Docker.md [hunk 12]
```diff
@@ -3,6 +3,8 @@ title: " Kompose Helps Developers Move Docker Compose Files to Kubernetes "
 date: 2017-08-10
 slug: kompose-helps-developers-move-docker
 url: /blog/2017/08/Kompose-Helps-Developers-Move-Docker
+author: >
+  Charlie Drage (Red Hat)
 ---
 _Editor's note: today's post is by Charlie Drage, Software Engineer at Red Hat giving an update about the Kubernetes project Kompose._
```
Comment (sftim) [inline content/en/blog/_posts/2017-08-00-Kompose-Helps-Developers-Move-Docker.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-08-00-Kubernetes-Meets-High-Performance.md [hunk 13]
```diff
@@ -3,6 +3,8 @@ title: " Kubernetes Meets High-Performance Computing "
 date: 2017-08-22
 slug: kubernetes-meets-high-performance
 url: /blog/2017/08/Kubernetes-Meets-High-Performance
+author: >
+   Robert Lalonde (Univa) 
 ---
 Editor's note: today's post is by Robert Lalonde, general manager at Univa, on supporting mixed HPC and containerized applications &nbsp;
```
Comment (sftim) [inline content/en/blog/_posts/2017-08-00-Kubernetes-Meets-High-Performance.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-09-00-Introducing-Resource-Management-Working.md [hunk 14]
```diff
@@ -3,6 +3,8 @@ title: " Introducing the Resource Management Working Group "
 date: 2017-09-21
 slug: introducing-resource-management-working
 url: /blog/2017/09/Introducing-Resource-Management-Working
+author: >
+   Jeremy Eder (Red Hat)
 ---
 _**Editor's note: today's post is by Jeremy Eder, Senior Principal Software Engineer at Red Hat, on the formation of the Resource Management Working Group**_
```
Comment (sftim) [inline content/en/blog/_posts/2017-09-00-Introducing-Resource-Management-Working.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-09-00-Kubernetes-Statefulsets-Daemonsets.md [hunk 15]
```diff
@@ -4,6 +4,9 @@ title: " Kubernetes StatefulSets & DaemonSets Updates "
 date: 2017-09-27
 slug: kubernetes-statefulsets-daemonsets
 url: /blog/2017/09/Kubernetes-Statefulsets-Daemonsets
+author: >
+   Janet Kuo (Google),
+   Kenneth Owens (Kenneth Owens)
 ---
 Editor's note: today's post is by Janet Kuo and Kenneth Owens, Software Engineers at Google.
```
Comment (sftim) [inline content/en/blog/_posts/2017-09-00-Kubernetes-Statefulsets-Daemonsets.md:11]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-09-00-Windows-Networking-At-Parity-With-Linux.md [hunk 16]
```diff
@@ -3,6 +3,8 @@ title: " Windows Networking at Parity with Linux for Kubernetes "
 date: 2017-09-08
 slug: windows-networking-at-parity-with-linux
 url: /blog/2017/09/Windows-Networking-At-Parity-With-Linux
+author: >
+   Jason Messer (Microsoft)
 ---
 _**Editor's note: today's post is by Jason Messer, Principal PM Manager at Microsoft, on improvements to the Windows network stack to support the Kubernetes CNI model.**_
```
Comment (sftim) [inline content/en/blog/_posts/2017-09-00-Windows-Networking-At-Parity-With-Linux.md:9]: I'd drop this note. We now just credit the person and optionally their organization.

File: content/en/blog/_posts/2017-10-00-Enforcing-Network-Policies-In-Kubernetes.md [hunk 17]
```diff
@@ -3,6 +3,8 @@ title: " Enforcing Network Policies in Kubernetes "
 date: 2017-10-30
 slug: enforcing-network-policies-in-kubernetes
 url: /blog/2017/10/Enforcing-Network-Policies-In-Kubernetes
+author: >
+   Ahmet Alp Balkan (Google) 
 ---
 _**Editor's note: this post is part of a [series of in-depth articles](https://kubernetes.io/blog/2017/10/five-days-of-kubernetes-18) on what's new in Kubernetes 1.8. Today’s post comes from Ahmet Alp Balkan, Software Engineer, Google.**_
```
Comment (sftim) [inline content/en/blog/_posts/2017-10-00-Enforcing-Network-Policies-In-Kubernetes.md:9]: I would do this:
```suggestion
_**Editor's note:** this post is part of a [series of in-depth articles](/blog/2017/10/five-days-of-kubernetes-18) on what's new in Kubernetes 1.8._
```

File: content/en/blog/_posts/2017-10-00-Kubeadm-V18-Released.md [hunk 18]
```diff
@@ -3,6 +3,8 @@ title: "  kubeadm v1.8 Released: Introducing Easy Upgrades for Kubernetes Cluste
 date: 2017-10-25
 slug: kubeadm-v18-released
 url: /blog/2017/10/Kubeadm-V18-Released
+author: >
+  Lucas Käldström (Weaveworks)
 ---
 **_Editor’s note: this post is part of a [series of in-depth articles](https://kubernetes.io/blog/2017/10/five-days-of-kubernetes-18) on what's new in Kubernetes 1.8_**
```
Comment (sftim) [inline content/en/blog/_posts/2017-10-00-Kubeadm-V18-Released.md:9]: I would do this:
```suggestion
_**Editor's note:** this post is part of a [series of in-depth articles](/blog/2017/10/five-days-of-kubernetes-18) on what's new in Kubernetes 1.8._
```

File: content/en/blog/_posts/2017-10-00-Request-Routing-And-Policy-Management.md [hunk 19]
```diff
@@ -3,6 +3,10 @@ title: " Request Routing and Policy Management with the Istio Service Mesh "
 date: 2017-10-10
 slug: request-routing-and-policy-management
 url: /blog/2017/10/Request-Routing-And-Policy-Management
+author: >
+  Frank Budinsky (IBM),
+  Andra Cismaru (Google),
+  Israel Shalom (Google)
 ---
  **_Editor's note: Today’s post by Frank Budinsky, Software Engineer, IBM, Andra Cismaru, Software Engineer, Google, and Israel Shalom, Product Manager, Google, is the second post in a three-part series on Istio. It offers a closer look at request routing and policy management._**
```
Comment (sftim) [inline content/en/blog/_posts/2017-10-00-Request-Routing-And-Policy-Management.md:11]: I suggest:
```suggestion
_**Editor's note:** Today’s post is the second post in a three-part series on Istio._
```

File: content/en/blog/_posts/2017-10-00-Software-Conformance-Certification.md [hunk 20]
```diff
@@ -3,6 +3,8 @@ title: " Introducing Software Certification for Kubernetes "
 date: 2017-10-19
 slug: software-conformance-certification
 url: /blog/2017/10/Software-Conformance-Certification
+author: >
+  William Denniss (Google)
 ---
 
 _**Editor's Note: Today's post is by William Denniss, Product Manager, Google Cloud on the new Certified Kubernetes Conformance Program.**_
```
Comment (sftim) [inline content/en/blog/_posts/2017-10-00-Software-Conformance-Certification.md:10]: I'd drop the note.

File: content/en/blog/_posts/2017-10-00-Using-Rbac-Generally-Available-18.md [hunk 21]
```diff
@@ -3,6 +3,8 @@ title: " Using RBAC, Generally Available in Kubernetes v1.8 "
 date: 2017-10-28
 slug: using-rbac-generally-available-18
 url: /blog/2017/10/Using-Rbac-Generally-Available-18
+author: >
+  Eric Chiang (CoreOS)
 ---
 **_Editor's note: this post is part of a [series of in-depth articles](https://kubernetes.io/blog/2017/10/five-days-of-kubernetes-18) on what's new in Kubernetes 1.8. Today’s post comes from Eric Chiang, software engineer, CoreOS, and SIG-Auth co-lead._**
```
Comment (sftim) [inline content/en/blog/_posts/2017-10-00-Using-Rbac-Generally-Available-18.md:9]: I would do this:
```suggestion
_**Editor's note:** this post is part of a [series of in-depth articles](/blog/2017/10/five-days-of-kubernetes-18) on what's new in Kubernetes 1.8._
```

File: content/en/blog/_posts/2017-11-00-Kubernetes-Easy-Way.md [hunk 22]
```diff
@@ -3,6 +3,8 @@ title: " Kubernetes the Easy Way "
 date: 2017-11-01
 slug: kubernetes-easy-way
 url: /blog/2017/11/Kubernetes-Easy-Way
+author: >
+  Dan Garfield (Codefresh)
 ---
  **_Editor's note: Today's post is by Dan Garfield, VP of Marketing at Codefresh, on how to set up and easily deploy a Kubernetes cluster._**
```
Comment (sftim) [inline content/en/blog/_posts/2017-11-00-Kubernetes-Easy-Way.md:9]: I'd drop the note.

File: content/en/blog/_posts/2017-11-00-Securing-Software-Supply-Chain-Grafeas.md [hunk 23]
```diff
@@ -3,6 +3,9 @@ title: " Securing Software Supply Chain with Grafeas "
 date: 2017-11-03
 slug: securing-software-supply-chain-grafeas
 url: /blog/2017/11/Securing-Software-Supply-Chain-Grafeas
+author: >
+  Kelsey Hightower (Google),
+  Sandra Guo (Google)
 ---
  **_Editor's note: This post is written by Kelsey Hightower, Staff Developer Advocate at Google, and Sandra Guo, Product Manager at Google._**
```
Comment (sftim) [inline content/en/blog/_posts/2017-11-00-Securing-Software-Supply-Chain-Grafeas.md:10]: I'd drop the note.

File: content/en/blog/_posts/2017-12-00-Introducing-Kubeflow-Composable.md [hunk 24]
```diff
@@ -3,6 +3,9 @@ title: " Introducing Kubeflow - A Composable, Portable, Scalable ML Stack Built
 date: 2017-12-21
 slug: introducing-kubeflow-composable
 url: /blog/2017/12/Introducing-Kubeflow-Composable
+author: >
+  Jeremy Lewi (Google),
+  David Aronchick (Google)
 ---
 
 **_Today’s post is by David Aronchick and Jeremy Lewi, a PM and Engineer on the Kubeflow project, a new open source GitHub repo dedicated to making using machine learning (ML) stacks on Kubernetes easy, fast and extensible._**
```
Comment (sftim) [inline content/en/blog/_posts/2017-12-00-Introducing-Kubeflow-Composable.md:11]: I'd drop the note.

File: content/en/blog/_posts/2017-12-00-Paddle-Paddle-Fluid-Elastic-Learning.md [hunk 25]
```diff
@@ -3,6 +3,14 @@ title: " PaddlePaddle Fluid: Elastic Deep Learning on Kubernetes "
...
+author: >
+  Xu Yan (Baidu Research),
+  Helin Wang (Baidu Research),
+  Yi Wu (Baidu Research),
+  Xi Chen (Baidu Research),
+  Weibao Gong (Baidu Research),
+  Xiang Li (CoreOS),
+  Yi Wang (Baidu Research) 
 ---
 _Editor's note: Today's post is a joint post from the deep learning team at Baidu and the etcd team at CoreOS._
```
Comment (sftim) [inline content/en/blog/_posts/2017-12-00-Paddle-Paddle-Fluid-Elastic-Learning.md:15]: I would do this:
```suggestion
_**Editor's note:** Today's post is a joint post from the deep learning team at Baidu and the etcd team at CoreOS_
```

**Review by sftim — COMMENTED** [review]
Thanks

LGTM, but I'd prefer some more edits.

**Comment by dipesh-rawat** [comment]
The latest commit push in the PR reflects the changes based on the review comment https://github.com/kubernetes/website/pull/46627#pullrequestreview-2092507724

**Pull request merged**